### PR TITLE
Adding plot cell activity functionality to TP mixin

### PR DESCRIPTION
**Pull request by rmccall84**
Added a method which displays a "stability plot" of active cells over time.
screen shot 2015-02-02 at 11 30 00 am

### CODE REVIEW

**Comment by rmccall84** [comment]
@scottpurdy this is the other half of the previous PR.

**Comment by scottpurdy** [comment]
@rmccall84 - a brief PR description would be nice, esp. if it had an image of a generated plot

File: sensorimotor/sensorimotor/temporal_pooler_monitor_mixin.py [hunk 1]
```diff
@@ -134,6 +134,23 @@ def mmGetPlotConnectionsPerColumn(self, title=None):
...
+    activeCellTrace = self._mmTraces["activeCells"].data
+    data = numpy.zeros((self._numColumns, 1))
+    for activeSet in activeCellTrace:
+      activity = numpy.zeros((self._numColumns, 1))
+      activity[list(activeSet)] = 1
+      data = numpy.concatenate((data, activity), 1)
+
+    plot.addImage(data, xlabel="Time", ylabel="Cell Activity")
+    return plot
+
```
Comment (scottpurdy) [inline sensorimotor/sensorimotor/temporal_pooler_monitor_mixin.py]: Missing a newline here (should be two blank lines in between)

**Comment by scottpurdy** [comment]
:+1:

**Comment by scottpurdy** [comment]
@rmccall84 - I'll merge once you resolve the one comment

**Comment by rmccall84** [comment]
@scottpurdy  I ran this by @chetan51 and added the reset lines. I chose to simply make the "background" of a particular set of active cells gray to show reset. Other options include 1) skipping that time step's data and having a solid bar,  or  2) adding a padding line which messes up the corresponds of the time steps to the data. Let me know if you prefer one of these other methods instead of this.

screen shot 2015-02-02 at 4 21 29 pm

**Comment by chetan51** [comment]
@rmccall84 I like this background idea, it fits well with how the tables print the data as well. I would keep it this way, thanks!

**Comment by scottpurdy** [comment]
Yup, background sounds good. I was thinking inserting a new line but since that messes up the x axis I support the background change. Looks nice!

**Pull request merged**